src/main/java/com/assure/movie/model/domain/Member.java

```diff
@@ -11,7 +11,7 @@
 public class Member implements Identifiable<Long> {
 
     @Id
-    @GeneratedValue(strategy= GenerationType.AUTO)
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Long id;
 
```

Comment:
This comment should be in line 31, but I cannot add nothig there!
You are defining this relation as a OneToOne, does this mean that a member will not be able to rent more than one movie?

Reply:
You're right that is my mistake. the Member and the RentalMovie entities must have had a relationship one to many.